src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -115,6 +118,23 @@ class WorkflowManagerActor(dataAccess: DataAccess, backend: Backend) extends Act
     }
   }
 
+  private def assertCallFqnWellFormed(callFqn: FullyQualifiedName): Future[(String, String)] = {
+    callFqn.split("\\.").toSeq match {
+      case s:Seq[String] if s.size >= 2 => Future.successful(s.head, s.last)
```

Comment:
I'm not sure what correct behavior is.  The backends will just use `call.name` to name directories, so I think `wf1.wf1.call_name` would still create a directory called `call-call_name`.  However we know this won't be scalable in scatter-gather, loops, and nested workflows.  Either I should tackle this now or we should file a ticket to tackle this while doing scatter/gather.